src/ObjectEncryptor.php

```diff
@@ -237,10 +237,38 @@ private function encryptItem($key, $value, array $wrappers, CryptoWrapperInterfa
         }
     }
 
+    private function getKnownWrapperPrefixes(): array
+    {
+        return [
+            (new GenericKMSWrapper())->getPrefix(),
+            (new GenericAKVWrapper())->getPrefix(),
+            (new ComponentKMSWrapper())->getPrefix(),
+            (new ComponentAKVWrapper())->getPrefix(),
+            (new ProjectKMSWrapper())->getPrefix(),
+            (new ProjectAKVWrapper())->getPrefix(),
+            (new ConfigurationKMSWrapper())->getPrefix(),
+            (new ConfigurationAKVWrapper())->getPrefix(),
+            // legacy wrappers that are no longer implemented, but still exist in the real world
+            'KBC::Encrypted==',
+            'KBC::ComponentEncrypted==',
+            'KBC::ComponentProjectEncrypted==',
+        ];
+    }
+
+    private function checkKnownWrapper(string $value): bool
```

Comment:
`isKnownWrapper` ?

Reply:
Hmno, tak ještě se můžou ty dva zacryptovany fieldy unsetnout, jestli by ti to přišlo lepší

Reply:
Co? Jaky fieldy?

Reply:
takhle